scripts/audit_frontend_licenses.py

```diff
@@ -169,15 +144,15 @@ def check_licenses(licenses) -> NoReturn:
         print(f"{len(bad_packages)} unacceptable licenses")
         sys.exit(1)
 
-    print(f"No unacceptable licenses")
+    print(f"No unacceptable licenses: {len(packages)} checked")
```

Comment:
nit: is the number of checked packages the same / similar as with the previous check?

Reply:
We are actually checking more licenses it seems 2336 vs 1164